test/services/GuidanceServiceSpec.scala

```diff
@@ -107,14 +107,14 @@ class GuidanceServiceSpec extends BaseSpec {
 
       MockPageRenderer
         .renderPage(pages(2), labels)
-        .returns((pages(2).stanzas, labels, None))
+        .returns((pages(2).stanzas.collect{case s: VisualStanza => s}, labels, None))
```

Comment:
Can we be consistent with the page definition; either pages(2) or pages.last

Reply:
last => pages(2)